src/AWS/Signers/V4.elm

```diff
@@ -1,13 +1,13 @@
 module AWS.Signers.V4 exposing (..)
 
 import AWS.Config exposing (Credentials)
+import AWS.Http exposing (RequestParams, UnsignedRequest)
 import AWS.Signers.Canonical exposing (canonical, signedHeaders)
-import AWS.Http exposing (UnsignedRequest, RequestParams)
 import Date exposing (Date)
-import Date.Format exposing (formatISO8601)
+import Date.Extra exposing (toUtcIsoString)
```

Comment:
`Date.Format.formatISO8601` returns the wrong value. It doesn't take into account the timezone. See https://github.com/mgold/elm-date-format/issues/18